log/log.go

```diff
@@ -133,3 +155,30 @@ func addLevelsToBackend(config *util.Config, prefix string, backend logging.Leve
 		backend.SetLevel(level, moduleName)
 	}
 }
+
+// Log formats.
+var (
+	DefaultFormat = "%{color}%{time:15:04:05.000}: %{module} %{level} %{color:reset} %{message}"
+	CliFormat     = "%{color}%{message}%{color:reset}"
+)
+
+//SetUpBasicLogging configures logging to output logs to w.
+func SetUpBasicLogging(w io.Writer, format string, modlevs ...interface{}) {
+	if len(modlevs)%2 != 0 {
+		panic("Invalid number of parameters")
+	}
+
+	backendFormatter := logging.NewBackendFormatter(
+		logging.NewLogBackend(w, "", 0),
+		logging.MustStringFormatter(format),
+	)
+	leveledBackendFormatter := logging.AddModuleLevel(backendFormatter)
+
+	for i := 0; i < len(modlevs); i += 2 {
```

Comment:
This looks strange for me. IMO a structure with a parameter and value would be better than assuming specific parameter types and specific order.